### PR TITLE
feat(va-testing.data-commons.org): test deployment on 20241028

### PR DESCRIPTION
**Pull request by m0nhawk**
Link to Jira ticket if there is one: [VADC-1496](https://ctds-planx.atlassian.net/browse/VADC-1496)

### Environments
* va-testing

### Description of changes
* deploying 2024.10
* data-portal `fix_enable_legacy_vs_new_data_dictionary_va` includes new data dictionary and 2024.10 release (no attrition modal, that'll be in 2024.11)
* 508 changes from prod https://github.com/uc-cdis/cdis-manifest/pull/8011

[VADC-1496]: https://ctds-planx.atlassian.net/browse/VADC-1496?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: va-testing.data-commons.org/manifest.json [hunk 1]
```diff
@@ -8,25 +8,25 @@
...
-    "dashboard": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-statics:2024.09",
-    "fence": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/fence:10.2.0",
+    "dashboard": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-statics:2024.10",
+    "fence": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/fence:2024.10",
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.15.3-debian-cloudwatch-1.0",
-    "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:2024.09",
-    "indexd": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexd:2024.09",
-    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2024.09",
-    "metadata": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/metadata-service:2024.09",
-    "ohdsi-atlas": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ohdsi-atlas:feat_enable_cohort_characterization",
```
Comment (pieterlukasse) [inline va-testing.data-commons.org/manifest.json:22]: maybe leave this one... it is being tested

File: va-testing.data-commons.org/manifest.json [hunk 2]
```diff
@@ -8,25 +8,25 @@
...
+    "ohdsi-atlas": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ohdsi-atlas:VA-pre-2024.08.19",
     "ohdsi-webapi": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ohdsi-webapi:VA-pre-2024.08.19",
-    "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2024.09",
-    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:5.33.1",
-    "revproxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/nginx:2024.09",
-    "sheepdog": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sheepdog:2024.09",
-    "ssjdispatcher": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ssjdispatcher:2024.09",
-    "wts": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/workspace-token-service:2024.09"
+    "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2024.10",
+    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:fix_enable_legacy_vs_new_data_dictionary_va",
```
Comment (pieterlukasse) [inline va-testing.data-commons.org/manifest.json:25]: let's merge to master and make a release

Comment (pieterlukasse) [reply va-testing.data-commons.org/manifest.json:25]: https://github.com/uc-cdis/data-portal/releases/tag/VA-pre-2024.10.25

**Review by nmetokishlubsky — APPROVED** [review]

File: va-testing.data-commons.org/manifest.json (hunk 1 above)
Comment (m0nhawk) [reply va-testing.data-commons.org/manifest.json:22]: I put it back. 👍

File: va-testing.data-commons.org/manifest.json (hunk 2 above)
Comment (m0nhawk) [reply va-testing.data-commons.org/manifest.json]: 👍

**Comment by pieterlukasse** [comment]
👍

**Pull request merged**